jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -370,6 +373,39 @@ public void testJibTaskGroupIsSet() {
         taskName -> Assert.assertEquals(taskName, "Jib", tasks.getByPath(taskName).getGroup()));
   }
 
+  @Test
+  public void testLazyEvalForImageAndTags() {
+    try {
+      testProject.build(JibPlugin.JIB_EXTENSION_NAME);
```

Comment:
I think this technically works, but is confusing. Maybe `JibPlugin.BUILD_IMAGE_TASK_NAME` is more appropriate?

Reply:
Ah that makes more sense